sanic/request.py

```diff
@@ -192,18 +191,9 @@ def socket(self):
         return self._socket
 
     def _get_address(self):
-        sock = self.transport.get_extra_info('socket')
-
-        if sock.family == socket.AF_INET:
-            self._socket = (self.transport.get_extra_info('peername') or
-                            (None, None))
-            self._ip, self._port = self._socket
-        elif sock.family == socket.AF_INET6:
-            self._socket = (self.transport.get_extra_info('peername') or
-                            (None, None, None, None))
-            self._ip, self._port, *_ = self._socket
-        else:
-            self._ip, self._port = (None, None)
+        self._socket = self.transport.get_extra_info('peername') or \
+                        (None, None)
+        self._ip, self._port = self._socket[0], self._socket[1]
```

Comment:
Just for the sake of clarity, since we're not unpacking values, can't we just do this:

```python
self._ip = self._socket[0]
self._port = self._socket[1]
```

Or, if a one liner _is a must_ (for unpacking), then this:

```python
self._ip, self._port = self._socket[:2]
```

... ?

Personally, I would vote for the first.

Reply:
I'm always a fan of clarity over compactness.

Reply:
@vltr fixed